boards/ct/ctcc_nrf52840/board.yml

```diff
@@ -0,0 +1,10 @@
+board:
+  name: ctcc_nrf52840
```

Comment:
the board name should be ctcc without nrf52840, then it can be used as a base board and future soc variants added in future